### ISSUE TITLE
Optimize Google Tag Manager on the 350.org website

### ISSUE THREAD

**Issue by cjmabry** [350org/ak_intl_v3#371]
@cjmabry can I leave it with you to share your updated recommendations on this one based on your doc + Bob's findings?

**Comment by cjmabry** [comment]
@rachelhbd I converted this to an issue so I could flesh it out but it might make sense to move to the Wordpress project on Github, although I do not think I have access to do that.

**Comment by rachelhbd** [comment]
Yep thanks, I think we could writeup here for now as our wordpress repo is in gitlab, but I believe @rezziemaven @stbensonimoh are likely to move it to github soon

**Comment by rachelhbd** [comment]
@rezziemaven do you think this would be worth looking at in September before we get into content testing? It was suggested by Chris/Bob to increase pagespeed

**Comment by cjmabry** [comment]
@rachelhbd I think it might make sense to tackle the testing things outlined here https://github.com/350org/action-tools/issues/19#issuecomment-1234674509 either before or in tandem with this, as we don't want to mess up any pages, but probably less of a hard requirement here.

**Comment by rachelhbd** [comment]
Tentatively moving to January if we consider these as pre-requisites (but open to moving faster!)

https://github.com/350org/action-tools/issues/38 - Nov
https://github.com/350org/action-tools/issues/39 - Dec

Also curious how this one relates/should be sequences - https://github.com/350org/action-tools/issues/19

**Comment by cjmabry** [comment]
Going to move this to icebook as I think we are pretty full up right now. cc @rachelhbd
